hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -151,80 +151,87 @@ prepareServerOutput ::
   -- | Final output
   LBS.ByteString
 prepareServerOutput ServerOutputConfig{txOutputFormat, utxoInSnapshot} response =
-  cookUtxoInclusion cookTxOutput
```

Comment:
@ffakenz can you undo the refactoring in commit 617c2a1e8c2ff0e0a59a0e26692d45b3b0a1a117? I would vote for the code before as it was more clear and separated the two purposes (maybe the word "cook" could be improved).